covid_xprize/examples/prescriptors/neat/prescribe.py

```diff
@@ -32,25 +44,54 @@ def prescribe(start_date_str: str,
     start_date = pd.to_datetime(start_date_str, format='%Y-%m-%d')
     end_date = pd.to_datetime(end_date_str, format='%Y-%m-%d')
```

Comment:
Previously, this file was ignoring the `prior_ips_file`, and simply pulling the ips from the historical data file. Now we actually use the `prior_ips_file`.